### DISCUSSION TITLE
eslint-config-next needs to bump eslint-plugin-import dependency version

### DISCUSSION THREAD

**Discussion by IDrumsey** [vercel/next.js#44472] [Help]
After running `npm audit`, I'm getting the following

```
# npm audit report

json5  <2.2.2
Severity: high
Prototype Pollution in JSON5 via Parse Method - https://github.com/advisories/GHSA-9c47-m6qq-7p4h
fix available via `npm audit fix --force`
Will install eslint-config-next@12.0.4, which is a breaking change
node_modules/tsconfig-paths/node_modules/json5
  tsconfig-paths  3.5.0 - 3.9.0 || 3.11.0 - 3.14.1
  Depends on vulnerable versions of json5
  node_modules/tsconfig-paths
    eslint-plugin-import  >=2.24.2
    Depends on vulnerable versions of tsconfig-paths
    node_modules/eslint-plugin-import
      eslint-config-next  >=12.0.5-canary.0
      Depends on vulnerable versions of eslint-plugin-import
      node_modules/eslint-config-next
```

This has to do with json5 which looks like it is fixed with @2.2.2

- tsconfig-paths has a PR waiting for merge
- eslint-import-plugin has an issue submitted - see https://github.com/import-js/eslint-plugin-import/issues/2634

Once those have been fixed, eslint-import-plugin will likely need bumped to integrate this fix.

**Comment by gigibot-tech** [comment]
any updates?

**Comment by allanncruz** [comment]
I'm using npm-check-updates. So far no updates

**Comment by IDrumsey** [comment]
I was reading somewhere this isn't a threat because you would have to attack yourself, but I don't know if that's true, plus it would be nice to not have these blaring issues when running audit.

**Comment by z33bs** [comment]
encountering same warnings my side

**Comment by CaptShmo** [comment]
also looking for an answer on this

**Comment by Youssef-Durgham** [comment]
me two

**Comment by codeaid** [comment]
GitHub's Dependabot also notified me about the vulnerability but it can't upgrade `json5` to `2.2.2` because:

> eslint-config-next@13.0.2 requires json5@^1.0.1 via a transitive dependency on tsconfig-paths@3.14.1

**Comment by madhavajay** [comment]
<img src="https://media2.giphy.com/media/39u4cdrGn47m4hbmhZ/giphy.gif" />

**Comment by YannikWeissenfels** [comment]
This vulnerability is patched in json5 v2.2.2 and later. A patch has also been backported for json5 v1 in versions v1.0.2 and later.
